src/App.jsx

```diff
@@ -1,5 +1,48 @@
+import React, { useEffect, useState } from 'react'; // нужно ли импортировать в каждый файл?
```

Comment:
please remove question comments
and yes it is ok to import in each file

Reply:
done